src/main/java/io/strimzi/test/container/StrimziKafkaCluster.java

```diff
@@ -37,17 +37,25 @@ public class StrimziKafkaCluster implements Startable {
     private final int brokersNum;
     private final Network network;
     private final StrimziZookeeperContainer zookeeper;
-    private final Collection<StrimziKafkaContainer> brokers;
+    private final Collection<KafkaContainer> brokers;
 
     /**
      * Constructor for @StrimziKafkaCluster class, which allows you to specify number of brokers @see{brokersNum},
-     * replication factor of internal topics @see{internalTopicReplicationFactor} and map of additional Kafka
-     * configuration @see{additionalKafkaConfiguration}.
+     * replication factor of internal topics @see{internalTopicReplicationFactor}, a map of additional Kafka
+     * configuration @see{additionalKafkaConfiguration} and a {@code proxyContainer}.
+     * <br><br>
+     * The {@code proxyContainer} allows to simulate network conditions (i.e. connection cut, latency).
+     * For example, you can simulate a network partition by cutting the connection of one or more brokers.
+     *
      * @param brokersNum number of brokers
      * @param internalTopicReplicationFactor internal topics
      * @param additionalKafkaConfiguration additional Kafka configuration
+     * @param proxyContainer Proxy container
      */
-    public StrimziKafkaCluster(final int brokersNum, final int internalTopicReplicationFactor, final Map<String, String> additionalKafkaConfiguration) {
+    public StrimziKafkaCluster(final int brokersNum,
```

Comment:
I guess this is a public interface which we are changing? Should we instead have another one for backwards compatibility?

Reply:
Yes, good point.